src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+    export let item;
+    let isOpen = false;
+</script>
+
+<div class="column" on:click={() => isOpen = !isOpen}>
+    <div class="row justify v-center">
+        <h5 class="body-3">{item.title}</h5>
+        <Svg id='arrow' w="16" />
+    </div>
+    {#if isOpen}
+        <div class="row column mrg-l mrg--t mrg--b desc">
```

Comment:
1. `row column` is not needed, since it doesn't change the behaviour of the `div` element;
2. Better to rename the `desc` class to be more descriptive, e.g. `description`